README.md

```diff
@@ -94,6 +94,7 @@ The options file is described in json (example in `specs/options.json`), and has
 |validateModelPropertiesExample|boolean|`true`, `false`|Ensures that the properties of the Schemas have an example value defined|
 |validateModelNoLocalDef|boolean|`true`, `false`|Not implemented yet|
 |validateNaming|boolean|`true`, `false`|Ensures the names follow a given naming convention|
+|ignoreHeaderXNaming|boolean|`true`, `false`|Exclude from validation header parameters starting with `x-`|
```

Comment:
oh good find! I think I added this one wayyyyyyyy later and never updated the documentation! Thanks for the PR!